### ISSUE TITLE
Question on Actors in "dotnet-architecture / eShopOnDapr" Project

### ISSUE THREAD

**Issue by Shawn-Fan** [dotnet-architecture/eShopOnDapr#21]
"Actors" applied in the "services/api" is pretty cool, any plan on the  "services/domain" ?

**Comment by amolenk** [comment]
You can consider the `OrderingProcessActor` the implementation of the domain business logic.

**Comment by Shawn-Fan** [comment]
I am sorry to response it too late. Maybe my question is not clear, like the "Ordering.Domain" in the "eshopOnContainers", do you have any plan to apply DDD & Actor for the "Ordering.Domain" sub-application?

**Comment by amolenk** [comment]
The version of eShop in this repo focuses on giving light-weight examples using the Dapr building blocks. That's why we changed the Order service to an Dapr actor implementation. When making that change, we've consolidated all ordering code into a single project, so the solution does not contain a separate Ordering.Domain project anymore.
There are no plans to change this any further. For DDD, the best example is still in the original eShopOnContainers repo.
